src/consensus/pbft/libbyz/client_proxy.h

```diff
@@ -281,7 +281,7 @@ bool ClientProxy<T, C>::send_request(
 
   if (threading::ThreadMessaging::thread_count > 1)
   {
-    threading::ThreadMessaging::thread_messaging.add_task<ExecuteRequestMsg>(
+    threading::ThreadMessaging::thread_messaging.add_task(
```

Comment:
Here and throughout the PR, removing this template as the type is deduced from `msg`.